README.md

```diff
@@ -1,17 +1,38 @@
-# Sample
+﻿# ABOUT
 
-This module is for fun and trying out new crazy and wacky things with the game. It houses a number of example blocks and UI screens.
+Sample.
 
-### Credits for images:
+This module is for fun and trying out new crazy and wacky things with the game. 
+It houses a number of example blocks and UI screens.
 
-WhiteStone: http://www.planetminecraft.com/texture_pack/new-start---by-loufisch/
+ 
```

Comment:
Remove this extra newline. Unless you know what you're doing, separate paragraphs by one single newline.

Reply:
Also, convert these two different lines into a single line.